src/test/rpc/Simulate_test.cpp

```diff
@@ -0,0 +1,989 @@
+                                           .getJson(JsonOptions::none);
+                        }
+                        else
+                        {
+                            metadata = result[jss::meta];
+                        }
+                        if (BEAST_EXPECT(
+                                metadata.isMember(sfAffectedNodes.jsonName)))
+                        {
+                            BEAST_EXPECT(metadata[sfAffectedNodes].size() == 1);
+                            auto node = metadata[sfAffectedNodes][0u];
+                            if (BEAST_EXPECT(
+                                    node.isMember(sfModifiedNode.jsonName)))
+                            {
+                                auto modifiedNode = node[sfModifiedNode];
+                                BEAST_EXPECT(
+                                    modifiedNode[sfLedgerEntryType] ==
+                                    "AccountRoot");
+                                auto finalFields = modifiedNode[sfFinalFields];
+                                BEAST_EXPECT(
+                                    finalFields[sfBalance] ==
+                                    "99999999999999990");
+                            }
+                        }
+                        BEAST_EXPECT(metadata[sfTransactionIndex] == 0);
+                        BEAST_EXPECT(
+                            metadata[sfTransactionResult] ==
+                            "tecNO_DST_INSUF_XRP");
+                    }
+                };
+
+            Json::Value tx;
+
+            tx[jss::Account] = env.master.human();
+            tx[jss::TransactionType] = jss::Payment;
+            tx[sfDestination] = alice.human();
+            tx[sfAmount] = "1";  // not enough to create an account
+
+            // test with autofill
+            testTx(env, tx, testSimulation);
+
+            tx[sfSigningPubKey] = "";
+            tx[sfTxnSignature] = "";
+            tx[sfSequence] = 1;
+            tx[sfFee] = "10";
+
+            // test without autofill
+            testTx(env, tx, testSimulation);
+
+            // TODO: check that the ledger wasn't affected
+        }
+    }
+
+    void
+    testSuccessfulTransactionMultisigned()
+    {
+        testcase("Successful multi-signed transaction");
+
+        using namespace jtx;
+        Env env(*this);
+        static auto const newDomain = "123ABC";
+        Account const alice("alice");
+        Account const becky("becky");
+        Account const carol("carol");
+        env.fund(XRP(10000), alice);
+        env.close();
+
+        // set up valid multisign
+        env(signers(alice, 1, {{becky, 1}, {carol, 1}}));
+
+        {
+            auto validateOutput = [&](Json::Value const& resp,
+                                      Json::Value const& tx) {
+                auto result = resp[jss::result];
+                checkBasicReturnValidity(result, tx, 5, "20");
```

Comment:
"20" - > please change to calculated value (baseFee * 2 or something)

Reply:
Fixed in [f1ac9b8](https://github.com/XRPLF/rippled/pull/5069/commits/f1ac9b8dc444105d8227afbb6089c9486c351500)